src/components/common/YourCompany/AssociationType.vue

```diff
@@ -235,7 +236,7 @@ export default class AssociationType extends Mixins(CommonMixin) {
 
   /** Reset association type value to original */
   resetAssociationType () {
-    this.setBusinessInformation(this.getEntitySnapshot.businessInfo)
+    this.setBusinessInformation(this.getOriginalBusinessInfo || { ...EmptyBusinessInfo })
```

Comment:
Unlikely but this could have saved "undefined" in the store instead of the initial values (per state-model.ts).